packages/money-market/src/components/transactions/borrow/GhoBorrowModalContent.tsx

```diff
@@ -0,0 +1,207 @@
+import { InterestRate } from "@aave/contract-helpers"
+import {
+  calculateHealthFactorFromBalancesBigUnits,
+  USD_DECIMALS,
+  valueToBigNumber,
+} from "@aave/math-utils"
+import {
+  Flex,
+  Separator,
+  Stack,
+  SummaryRow,
+} from "@galacticcouncil/ui/components"
+import { bigShift } from "@galacticcouncil/utils"
```

Comment:
why do we use two different functions for shifting decimals. we already how one scale/scaleHuman. We need leave only one